### ISSUE TITLE
google marking sonarr running on my server as phishing

### ISSUE THREAD

**Issue by samueljim** [Sonarr/Sonarr#5919]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

This is the second time this happened to me. Google search console has decided that my centre instance which is exposed to the Internet is a fishing site and that's my domain should be flagged as insecure is the result of sonarr

![image](https://github.com/Sonarr/Sonarr/assets/19277887/eadf9af6-d185-4b2b-acf3-babdee59b9f1)

I have changed the domain using inspect element for privacy r

### Expected Behavior

That google won't flag this.
I have radarr running on the same server and don't have issues there 

### Steps To Reproduce

I've seen other people have this issue but i don't understand this 
https://www.reddit.com/r/Overseerr/comments/140hi6q/my_overseerr_instant_got_my_domain_flagged_on/

_No response_

### Environment

```markdown
- OS: windows
- Sonarr: 3.0.9.1555
- Docker Install: no
- Using Reverse Proxy: yes nginx
- Browser: arc
```

### What branch are you running?

Develop

### Trace Logs?

![image](https://github.com/Sonarr/Sonarr/assets/19277887/42a96888-8cc1-4cbf-8be1-b71cbfc8058c)

**Comment by markus101** [comment]
They're flagging a login page as suspicious, nothing actually suspicious with that. If they can provide more details explaining exactly what's wrong with the login page we could see what we could do to avoid that, but sounds more like Google needs to address their heuristics for that detection.

**Comment by markus101** [comment]
Will reopen if something actionable arises.
